tests/test_datasets/test_dataset.py

```diff
@@ -99,6 +107,58 @@ def test_get_data_with_ignore_attributes(self):
         self.assertEqual(len(categorical), 38)
         # TODO test multiple ignore attributes!
 
+    def test_get_data_numpy(self):
+        data = self.titanic.get_data(dataset_format='array')
+        self.assertTrue(isinstance(data, np.ndarray))
+        self.assertEqual(data.shape[1], len(self.titanic.features))
+        self.assertEqual(data.shape[0], 1309)
+        self.assertTrue(data.dtype == 'object')
+
+        X, y = self.titanic.get_data(
+            dataset_format='array',
+            target=self.titanic.default_target_attribute)
+        self.assertTrue(isinstance(X, np.ndarray))
+        self.assertTrue(isinstance(y, np.ndarray))
+        self.assertEqual(X.shape, (1309, 13))
+        self.assertEqual(y.shape, (1309,))
+        self.assertTrue(X.dtype == 'object')
+        self.assertTrue(y.dtype == int)
+
+    def test_get_data_pandas(self):
+        data = self.titanic.get_data(dataset_format='dataframe')
+        self.assertTrue(isinstance(data, pd.DataFrame))
+        self.assertEqual(data.shape[1], len(self.titanic.features))
+        self.assertEqual(data.shape[0], 1309)
+        col_dtype = {
+            'pclass': 'float64',
```

Comment:
Shouldn't this be an integer?

Reply:
Apparently it has been uploaded as numeric which is real

Reply:
As previously mentioned. It would be a numeric field.